acceptance/setup/aio/010_Install.rb

```diff
@@ -12,6 +12,15 @@ def install_from_local(hosts, package)
       scp_to(agent, package, tmpdir)
     end
 
+    if agent[:platform].match(/(?:el-7|redhat-7)/)
+      step "Upgrade openssl package...(" + agent[:platform] + ")" do
+      end
+      on(agent, "yum -y install openssl-1.0.1e-51.el7_2.4.x86_64")
```

Comment:
Is there any way we can avoid hard-coding this? Is it an upgrade or downgrade?

At least make it overridable with an environment variable.

Reply:
Yes that has been my intent/preference as well. I just wanted to get it to run thru CI successfully once and that time I was not sure if any env variables set at the time of launching the Step1 would be visible to any downstream steps (which needed some experimentation). In any case will check and do it if it possible.